src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -88,12 +100,15 @@ public TestCase persistTestCase(
     }
 
     TestCase enrichedTestCase = enrichNewTestCase(testCase, username);
+    verifyUniqueTestCaseName(enrichedTestCase, measure);
```

Comment:
maybe this can be done before `enrichedTestCase(...)` to avoid unnecessary processing in case of uniqueness violation.

Reply:
Admittedly, it was to solve a problem that doesn't exist. I was worried the enrichment method might be updated to modify some part of the Test Case name and we'd forget to move the verification check.

Of course, thinking about for 5 seconds, I realize the unit tests should catch that. 

Updating.